src/pages/home/report/ReportActionsView.js

```diff
@@ -155,9 +156,27 @@ function ReportActionsView(props) {
         if (oldestReportAction.actionName === CONST.REPORT.ACTIONS.TYPE.CREATED) {
             return;
         }
-
         // Retrieve the next REPORT.ACTIONS.LIMIT sized page of comments
-        Report.readOldestAction(reportID, oldestReportAction.reportActionID);
+        Report.getOlderActions(reportID, oldestReportAction.reportActionID);
+    };
+
+    /**
+     * Retrieves the next set of report actions for the chat once we are nearing the end of what we are currently
+     * displaying.
+     */
+    const loadNewerChats = ({distanceFromStart}) => {
+        // Only fetch more if we are not already fetching so that we don't initiate duplicate requests.
+        if (props.report.isLoadingNewerReportActions || props.report.isLoadingInitialReportActions) {
+            return;
+        }
+
+        // ideally we do not need use distanceFromStart here but due maxToRenderPerBatch and windowSize we receive a lot of renders so          times we can se how loadNewerChats is called. we use CONST.CHAT_HEADER_LOADER_HEIGHT to prevent this
```

Comment:
> lot of renders so          times

Typo ... what did you mean here?

Reply:
What I meant to say is that the component undergoes frequent re-renders due to the low value set for `maxToRenderPerBatch`. This leads to the 'onStartReached' callback being triggered multiple times.

To address this, we use `CONST.CHAT_HEADER_LOADER_HEIGHT` as a threshold to prevent unnecessary triggering of `onStartReached` when the chat is initially opened or when the user has reached the end of the list but hasn't scrolled further.

Additionally, we use throttling to the `onStartReached` callback to minimize its frequent invocation further.
This would be removed once the issue of frequent re-renders is resolved.